FrameworkSystem/Client/BundleDeliveryClient.py

```diff
@@ -77,13 +101,23 @@ def syncDir(self, bundleID, dirToSyncTo):
     self.log.info("Synchronizing dir with remote bundle")
     with tarfile.open(name='dummy', mode="r:gz", fileobj=buff) as tF:
       for tarinfo in tF:
-        tF.extract(tarinfo, dirToSyncTo)
+        try:
+          tF.extract(tarinfo, dirToSyncTo)
+        except OSError as e:
+          if e.errno != errno.EROFS:
+            raise e
```

Comment:
I think it's better not to check the `errno` as different file systems report this very differently:
```suggestion
```

If we really want to check it, it should check at least `EACCES`, `EPERM` and `EROFS`.

Reply:
Yes, you right..